react/ui-components/src/molecules/CustomDropdown.js

```
@@ -7,23 +7,23 @@ import Dropdown from "../atoms/Dropdown";
     master["filter"] = config?.mdmsConfig?.filter;
   }
 
   const { isLoading, data } = window?.Digit?.Hooks.useCustomMDMS(Digit?.ULBService?.getStateId(), config?.mdmsConfig?.moduleName, [master], {
     select: config?.mdmsConfig?.select
-      ? createFunction(config?.mdmsConfig?.select)
-      : (data) => {
+    ? createFunction(config?.mdmsConfig?.select)
+    : (data) => {
         const optionsData = _.get(data, `${config?.mdmsConfig?.moduleName}.${config?.mdmsConfig?.masterName}`, []);
         return optionsData
           .filter((opt) => (opt?.hasOwnProperty("active") ? opt.active : true))
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)